.eslintrc.json

```diff
@@ -4,7 +4,7 @@
     "es2021": true,
     "node": true
   },
-  "ignorePatterns": ["dist", "src/pinecone-generated-ts-fetch"],
```

Comment:
Needed to add this here bc for some reason Vercel was reading the client's linting config instead of the cloned repo's.

Reply:
(eventually, when we use what we learned here to build our own, lightweight version of a Vercel app for our Edge testing, we'll remove this, since we won't be cloning someone else's Vercel project)

Reply:
I think this is fine for now also, we don't need to worry about their linting if we're just cloning things.